ssh-server-config/set-login-grace-time.yaml

```diff
@@ -0,0 +1,64 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ssh-server-config
+  namespace: kube-system
+  labels:
+    app: ssh-server-config
+spec:
+  selector:
+    matchLabels:
+      app: ssh-server-config
+  template:
+    metadata:
+      labels:
+        app: ssh-server-config
+    spec:
+      hostPID: true
+      initContainers:
+        - name: ssh-server-config
+          image: gke.gcr.io/debian-base:bookworm-v1.0.3-gke.0@sha256:91b29592ee0b782c0ab777bfcabd14a0ae83d8e8eb90d3f0eb500acafae3f4e5
+          securityContext:
+            privileged: true
+          command:
+            - /bin/sh
+            - -c
+            - |
+              set -e
+              set -u
+              if [ ! -e "/etc/ssh/sshd_config" ] ; then
+                echo "/etc/ssh/sshd_config not found"
+                exit 1
+              fi
+
+              cp /etc/ssh/sshd_config /etc/ssh/sshd_config.cp
+              if grep -q "^LoginGraceTime" "/etc/ssh/sshd_config.cp"; then
+                  # Update existing LoginGraceTime
+                  sed -i "s/^LoginGraceTime.*/LoginGraceTime 0/" "/etc/ssh/sshd_config.cp"
+              else
+                  # Add new LoginGraceTime
+                  echo "LoginGraceTime 0" >> "/etc/ssh/sshd_config.cp"
+              fi
+
+              cp /etc/ssh/sshd_config.cp /etc/ssh/sshd_config
+              rm /etc/ssh/sshd_config.cp
+
+
+              EXEC="nsenter -t 1 -m -p --"
+              $EXEC systemctl restart sshd
```

Comment:
Do this need sshd restart or is systemctl reload sufficient? (Which should send SIGHUP and reload the config in place without restart?) xref: https://askubuntu.com/a/1070136